ember-autofocus-modifier/src/modifiers/autofocus.ts

```diff
@@ -0,0 +1,71 @@
+import { modifier } from "ember-modifier";
+import { next } from "@ember/runloop";
+
+const focusableElements = [
+  "BUTTON",
+  "SUMMARY",
+  "IFRAME",
+  "INPUT",
+  "SELECT",
+  "TEXTAREA",
+] as const;
```

Comment:
Have you looked into converting this to a string enum?

Reply:
Hey @fabmiz , yes! Since we're never referring to these values in other parts of the code, to me makes more sense to keep it as `const` (readonly) array. Also [enums are compiled to actual JS objects](https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-runtime) and we don't need it here.

Reply:
sounds good! TS [guides](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums) now suggests what you did.

> In modern TypeScript, you may not need an enum when an object with as const could suffice

Reply:
Yeah, I think that in some cases makes absolutely sense to use them. Nice to see that anyway 👍